base/stacktraces.jl

```diff
@@ -201,15 +201,19 @@ function show_spec_linfo(io::IO, frame::StackFrame)
 end
 
 function show(io::IO, frame::StackFrame; full_path::Bool=false)
-    print(io, " in ")
+    isreplerror = get(io, :REPLError, false)
+    print(io, isreplerror ? " ─ " : " in ")
```

Comment:
This feels unfortunate. I think we could add a second kwarg `prefix = " in "` instead, since the only place this is called with `REPLError = true` is `show_backtrace -> show_trace_entry`?